distribution/lib/Standard/Base/0.0.0-dev/src/System/File/Existing_File_Behavior.enso

```diff
@@ -44,7 +44,7 @@ type Existing_File_Behavior
 
        The `action` may not be run at all in case the `Error` behavior is
        selected.
-    write : File -> (Output_Stream -> Nothing) -> Nothing ! File_Error
+    write : File -> (Output_Stream -> Any) -> Any ! File_Error
```

Comment:
There is a lot more changes than just "Ensure that warnings are preserved on Nothing values passing back to Enso through polyglot boundary" - wouldn't it be better to split this into two pull requests?